### ISSUE TITLE
Overhaul generator creates random invalid blocks

### ISSUE THREAD

**Issue by ThizThizzyDizzy** [ThizThizzyDizzy/nc-reactor-generator#180]
Random shields, irradiators, etc. will be created not connected to anything during generation

**Comment by Gsonz04** [comment]
Yep, the calculator is useless for overhaul reactors. Generated reactors are full of invalid blocks.
